### DISCUSSION TITLE
Add more support for IPv6

### DISCUSSION THREAD

**Discussion by haraldkoch** [onedr0p/cluster-template#1510] [Ideas]
Long time user, first time caller.

I think there are some templating modifications that could be made to add more IPv6 support.

**Comment by onedr0p** [comment]
I think this part of this issue is blocked by Cilium: https://docs.cilium.io/en/latest/network/l2-announcements/

> Since this feature has no IPv6 support yet, only ARP messages are sent, no Unsolicited Neighbor Advertisements are sent.

Ref: https://github.com/cilium/cilium/issues/28985

Node IPv6 and kube-vip _should_ be possible. https://github.com/kube-vip/kube-vip/issues/279
